core-bundle/contao/templates/twig/backend/reload.stream.html.twig

```diff
@@ -0,0 +1,5 @@
+<turbo-stream action="update" target="top">
```

Comment:
this only works as long as `<body id="top">`, right?

Reply:
Yes, we need some identifier - according to the Turbo docs, it could also be a CSS query selector, but not sure how the detection for that is built. Maybe `target="html > body"` works…

 See https://turbo.hotwired.dev/reference/streams#targeting-multiple-elements for reference.

Reply:
In that case you need to use the `targets` (plural) attribute though. As far as I understand it, the `target` attribute only supports CSS ID targets.

Reply:
Yes, that works (see b7f64f2dc9cf49b3f967a728f6180b2d896aac0e).

In fact, it previously worked by accident because I forget the `<template>` tag which made Turbo somehow execute the whole thing anyways.